magento/adobe-stock-integration
# 1336: async run image files and content synchronization

### Description (*)
This PR in the current state is a POC for the image synchronization running async and will change in the final state.

### Fixed Issues (if relevant)
#1336

### Related Pull Requests
https://github.com/magento-engcom/magento2-infrastructure/pull/147


### Preconditions (*)

Install Magento and Adobe Stock Integration
Setup Magento Cron jobs: https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-cron.html

### Manual testing scenarios (*)

2. Upload a couple of images to the media gallery
3. Insert images to the category, product, page and blocks
4. Clear `media_gallery_asset` and `media_content_asset` tables 
5. Enable Enhanced Media Gallery in the configuration
6. Ensure the images are present in the enhanced media gallery and "Used In" block in image details is displaying the correct information
